app/views/layouts/_environmental_warning.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if ENV["REVIEW_APP"] == "true" %>
```

Comment:
I still don't see this -- at least on the mobile version. Why not? Is it a mobile thing?